### PR TITLE
Adds foam force to the loot tables in maintenance

### PR DESCRIPTION
**Pull request by ghost**
:cl: IT'S NERF OR NOTHING
add: Nanotrasen would like to remind you that all foam force weapons leftover from the great foam wars of 2540 are to be immediately handed over to security personnel...
/:cl:

image
yes the name is meant to be IT'S NERF OR NOTHING

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
>l6 saw foam in maint hmmmmmmmmmmmmmmmmmmmmm

**Comment by iskyp** [comment]
i see nothing wrong with assistants running around with l6 saws

**Comment by Euphausiacea** [comment]
Greytide powercreep disguised as qol

**Comment by iskyp** [comment]
>disguised as qol

what

**Comment by Rinoka** [comment]
for people who don't understand QoL: quality of life is reduction of annoyances/improvement of game flow, NOT "i have this little gimmicky fun thing i added".

this pr was never qol and i will suplex anyone who incorrectly calls shit qol. if you're going to insult a PR at least look like you have some idea what the fuck you're talking about

**Comment by iskyp** [comment]
he's not wrong about it being greytide powercreep though

**Comment by Armhulen** [comment]
@PKPenguin321 becoming lol

File: code/_globalvars/lists/maintenance_loot.dm [hunk 1]
```diff
@@ -104,5 +104,11 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
 	/obj/item/storage/secure/briefcase = 3,
 	/obj/item/storage/toolbox/artistic = 2,
 	/obj/item/toy/eightball = 1,
+	/obj/item/gun/ballistic/automatic/toy = 2,
```
Comment (ma44) [inline code/_globalvars/lists/maintenance_loot.dm]: These surprisingly don't have firing pins started in them I believe, use /obj/item/gun/ballistic/automatic/toy/unrestricted
from https://github.com/tgstation/tgstation/blob/057aa31cda6eb47505bc9fdd263fd1427b1276ed/code/modules/projectiles/guns/ballistic/toy.dm#L17

**Comment by ma44** [comment]
Also while this is a really long list and the chance of these not spawning in at all, the C20r, L6 Saw and the automatic/toy (AKA the saber SMG) could probably be a problem which I talked about at one point at https://github.com/tgstation/tgstation/pull/30485#issuecomment-327878401 
Points 4, 5 and 6 are probably valid.

_inb4 balance wise no but who cares y y y y_

**Comment by AlexTheSysop42** [comment]
wow, and I thought the C20r was the only one to worry about

File: code/_globalvars/lists/maintenance_loot.dm (hunk 1 above)
Comment (iskyp) [reply code/_globalvars/lists/maintenance_loot.dm]: the thing is, they do have firing pins, the unrestricted versions of non syndicate foam force are fucking pointless as they have the same firing pin
going to remove the pointless ones

File: code/_globalvars/lists/maintenance_loot.dm [hunk 2]
```diff
@@ -104,5 +104,11 @@ GLOBAL_LIST_INIT(maintenance_loot, list(
...
+	/obj/item/gun/ballistic/automatic/toy/pistol = 2,
+	/obj/item/gun/ballistic/shotgun/toy = 2,
+	/obj/item/gun/ballistic/shotgun/toy/crossbow = 2,
+	/obj/item/gun/ballistic/automatic/c20r/toy/unrestricted = 2,
+	/obj/item/gun/ballistic/automatic/l6_saw/toy/unrestricted = 1,
```
Comment (Qustinnus) [inline code/_globalvars/lists/maintenance_loot.dm]: lol ok

Comment (iskyp) [reply code/_globalvars/lists/maintenance_loot.dm]: >i see nothing wrong with assistants running around with l6 saws

**Comment by Cyberboss** [comment]
Good meme, but this isn't interesting enough to justify the chaos it'll cause imo

**Comment by iskyp** [comment]
i could gimp it to where it only spawns low tier foam force like shotguns and pistols @Cyberboss

**Pull request closed without merging**